### PR TITLE
chore: added ESLint as part of PR jobs

### PR DESCRIPTION
**Pull request by IgnatG**
### Changelog
- added ESLint as a required step on PR jobs

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-node@v3
+      with:
+        node-version: '18'
+        cache: 'npm'
+    - name: Install modules
+      run: npm ci
+    - name: Run ESLint (TypeScript)
+      run: npm run lint:fix
```
Comment (abhi-markan) [inline .github/workflows/lint.yml:18]: Should this not be `npm run lint` `fixing` lint changes in the pipeline will not be written to the repo.

**Review by abhi-markan — COMMENTED** [review]
Do we also need `.eslintrc.json`?

File: .github/workflows/lint.yml (hunk 1 above)
Comment (IgnatG) [reply .github/workflows/lint.yml:18]: that's true. I was testing it to see if it changes is in the pipeline - but it doesn't. So I'm taking it out

**Comment by IgnatG** [comment]
> Do we also need `.eslintrc.json`?

we have `.eslintrc.js`

**Review by abhi-markan — APPROVED** [review]

**Pull request merged**